draft-ietf-core-href.md

```diff
@@ -528,7 +528,8 @@ an absolute CRI reference:
 
 2. Initialize a buffer with the sections from the base CRI.
 
-3. If the value of discard is `true` in the CRI reference, replace the
+3. If the value of discard is `true` in the CRI reference (which is
+   implicitly the case when scheme and/or authority are present in the reference), replace the
```

Comment:
What RFC3986 calls a "network-path reference" (i.e., `"//" authority path-abempty`) in CRI terms should be expressed as `[ null, authority, path / null ]`, right?

If so, we could say:
```suggestion
3. If the value of discard is `true` in the CRI reference (which is
   implicitly the case when scheme and authority are present in the reference and scheme is null), replace the
```

Reply:
Drop this.  As discussed, I had misunderstood the whole thing.